stories/shellbar/shellbar.stories.js

```diff
@@ -36,8 +36,6 @@ The shellbar handles layout and has some built-in elements but relies on standal
 
 * <code>.fd-shellbar__subtitle</code> (optional) displays an application context. _This should be used rarely._
 
-* <code>.fd-product-menu</code> (optional) for navigating to applications within the product.
```

Comment:
This should be replaced with fd-shellbar__product-menu , not removed completely

Reply:
Actually we don't need this class anymore. I removed it from the shellbar